test/brod_topic_subscriber_SUITE.erl

```diff
@@ -388,7 +388,7 @@ check_state_continuity(Trace) ->
 
 %% Check that for any `init' there is a `terminate':
 check_init_terminate(Trace) ->
-  ?strict_causality( #{worker_id := _ID, ?snk_kind := topic_subscriber_init}
+  ?strict_causality( #{worker_id := _WorkerID, ?snk_kind := topic_subscriber_init}
```

Comment:
Was the intention to assert same ID in the two traces?

Reply:
Reverted. At some point (I think on another branch) I was getting a lint error. I wasn't able to reproduce it, so I just reverted it.